pdb-alignment-api/src/main/java/org/cbioportal/pdb_annotation/web/controllers/AlignmentController.java

```diff
@@ -49,15 +46,11 @@ public AlignmentController(AlignmentRepository alignmentRepository, EnsemblRepos
     @RequestMapping(value = "/StructureMappingQuery", method = RequestMethod.GET, produces = "application/json")
     @ResponseBody
     @ResponseStatus(HttpStatus.OK)
-    public List<Alignment> getPdbAlignmentByEnsemblId(
+    public List<Alignment> getPdbAlignmentByEnsemblId (
             @RequestParam @ApiParam(value = "Input ensembl id. For example ENSP00000483207.2", required = true, allowMultiple = true) String ensemblId) {
-        List<Alignment> list = new ArrayList();
-        try {
-            List<Alignment> it = alignmentRepository.findByEnsemblId(ensemblId);
-            list.addAll(it);
-        } catch (Exception ex) {
-            ex.printStackTrace();
-        }
+        List<Alignment> it = alignmentRepository.findByEnsemblId(ensemblId);
```

Comment:
I wondered about this too. I think the copy is not necessary ... deleting it now and updating PR